.github/workflows/pipeline.yml

```diff
@@ -45,6 +45,9 @@ jobs:
       - name: Check format
         run: npm run format:check
 
+      - name: Check autogenerated docs
+        run: npm run docs:gen && git diff --exit-code
```

Comment:
This command has been removed from `scripts` under package.json since it doesn't really make sense to be run locally, as any other change pending to be committed can make this check fail.